make_site.py

```diff
@@ -339,15 +344,19 @@ def download_N_theorems(kind: NTheorems) -> dict:
     if DOWNLOAD:
         download(f'https://leanprover-community.github.io/mathlib4_docs/{fname}', DATA/fname)
         with (DATA/fname).open('r', encoding='utf-8') as h_file:
-            n_theorems = [Type(thm,**content) for (thm,content) in yaml.safe_load(h_file).items()]
+            n_theorems = [Type(thm, **content) for (thm, content) in yaml.safe_load(h_file).items()]
             theorems = []
             for h in n_theorems:
                 assert not (h.decl and h.decls)
+                assert not h.statement and (h.decl or h.decls)
```

Comment:
```suggestion
                assert not (hasattr(h, 'statement') and h.statement) and (h.decl or h.decls)
```

Reply:
I've taken a look and decided to allow the statement yield in `100.yaml` instead: this way, we can avoid all the getattr dance.